### ISSUE TITLE
Hard-coded edmonton weather location

### ISSUE THREAD

**Issue by Accelsnow** [willasm/vc-weather#14]
The primary location set in vc-weather plugin settings do not seem to change the location of the weather displayed in both the status bar and the html embed. For example:

I set the primary location to be Sapporo (city in Japan):
![image](https://github.com/user-attachments/assets/313e1601-00a6-4034-89bb-efa62c21ba56)

My status bar template:

```
🔸%address% %month2-now%-%date2-now% %dow1-now% 🔸%temp% (%feelslike%) @ [%tempmin-today%, %tempmax-today%]🔸%conditions% | %preciptype% %precipprob%🔸
```

Status bar display:
![image](https://github.com/user-attachments/assets/03e3e222-c5be-47c6-80cb-5a4e619f3073)

The blizzard warning here takes me to [https://weather.gc.ca/](https://weather.gc.ca/), which is impossible since Sapporo is a Japanese city. When I hover over the warning, it shows that the warning should have expired on Nov 6.

![image](https://github.com/user-attachments/assets/dc2c40ba-8989-4f59-883d-c623cc9ac0db)

When I click on the weather status bar, the weather details displayed:
![image](https://github.com/user-attachments/assets/1ac0b4e5-8a13-4ec2-9ee2-aec79e49afd7)

The date is Aug 22 and city is edmonton.

My HTML embed template:

```
Weather
<img src=%iconurl-today% width="50" height="50"><br/>
%dow1-now% %year2-now%-%month2-now%-%date2-now%🔸%address%<br/> <b>%conditions-today%</b>🔸%temp%C (%feelslike%C) @ [<b><font color="#40E0D0">%tempmin-today%C</font></b>, <b><font color="#DE3163">%tempmax-today%C</font></b>]<br/>VIS %visibility%km🔸UV %uvindex%🔸%preciptype-today% %precipcover-today%<br/><u><b>D2</b></u>🔸%conditions-in1day%🔸[<font color="#40E0D0">%tempmin-in1day%C</font>, <font color="#DE3163">%tempmax-in1day%C</font>]<br/><u><b>D3</b></u>🔸%conditions-in2days%🔸[<font color="#40E0D0">%tempmin-in2days%C</font>, <font color="#DE3163">%tempmax-in2days%C</font>]
```

and it expands to:

![image](https://github.com/user-attachments/assets/d3a04cf1-f1bf-476d-8fd0-f0507b33de3e)

which also has edmonton displayed as the location.

Obsidian 1.7.5 and 1.7.6
vc-weather 0.3.1

**Comment by willasm** [comment]
So sorry about that, I had my testData.json file enabled. Will upload the correct version right away..

**Comment by willasm** [comment]
I just re-uploaded release v0.3.1 and removed the incorrect upload. You will need to force Brat to re-download the fixed version. Easiest way is to edit the file `manifest.json` and edit the line `"version": "0.3.1",` to `"version": "0.3.0",`.  That will cause Brat to re-download it. The file is located in this plugins folder `.obsidian/plugins/visual-crossing-weather`

While working on this plugin I use a local copy of the weather data. That prevents me from going over the daily limit for the api calls, which I have had happen a few times. Calling the api everytime I do a new build can use up the daily points quota very quickly. First time I have forgotten to diable it, So sorry about that..

**Comment by Accelsnow** [comment]
> I just re-uploaded release v0.3.1 and removed the incorrect upload. You will need to force Brat to re-download the fixed version. Easiest way is to edit the file `manifest.json` and edit the line `"version": "0.3.1",` to `"version": "0.3.0",`. That will cause Brat to re-download it. The file is located in this plugins folder `.obsidian/plugins/visual-crossing-weather`
> 
> While working on this plugin I use a local copy of the weather data. That prevents me from going over the daily limit for the api calls, which I have had happen a few times. Calling the api everytime I do a new build can use up the daily points quota very quickly. First time I have forgotten to diable it, So sorry about that..

That's what I thought it must have been some test config being unintentionally released 😂 Can confirm it is fixed by forcing a BRAT update (v0.3.1).
